warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java

```diff
@@ -7891,7 +7891,13 @@ public static String buildSelector(GeoTimeSerie gts) {
   
   public static String buildSelector(Metadata metadata) {
     StringBuilder sb = new StringBuilder();
+
+    char nameFirstChar = metadata.getName().charAt(0);
+    if ('=' == nameFirstChar || '~' == nameFirstChar) {
+      sb.append("%3D"); // Prepend '=' for the special character not to be interpreted
```

Comment:
Why add it as %3D instead of '='?

Reply:
It was to be consistent with the escaping of name and labels but I realize it is unnecessary because '=', '{' and '}' are not escaped.